### ISSUE TITLE
I'm stopping development of this project

### ISSUE THREAD

**Issue by XInTheDark** [XInTheDark/mzbot-v2#7]
MZ Bot has been amazing. I have developed it for more than a year now, and all sorts of useful features have been added. I also thank the Discord community for actively using and providing feedback on my bot.

However, I now announce that I will no longer be actively developing the bot.

Not that there is anything in particular that I hate. I am stopping development for a variety of reasons, for example:

1. I do not have so much time to work on this bot now.
2. I believe that the bot is almost "complete", and there are not many new features to be added. Despite that, I recognise that there are many bugs to be fixed and features to be enhanced, but those are no longer a top priority for me.
3. I do not agree with the general direction of how Discord is developing the bot system (see [this thread by Rapptz, the creator of discord.py](https://gist.github.com/Rapptz/4a2f62751b9600a31a0d3c78100287f1)). Discord is currently strongly encouraging every bot developer to switch to slash commands. In the future, it is highly probable that they will _force_ everyone to use slash commands; However, it is clear to me that the slash commands system was not developed properly. For example, using a slash command is around 5 times slower than using normal text commands: you have to first type the command, wait a split second for all the commands to load, choose the command you want, then enter parameters, and finally wait for the bot to respond. 
4. I do not have the time and patience to migrate to slash commands, and I think that it is useless to do so. Everyone on Discord is much more familiar with text commands.

To clarify: I don't think slash commands are a _bad_ idea in general, but the way Discord has implemented slash commands shows that they have not put much thought and effort into it.

Open source projects are a great idea; I will probably be creating them and/or be contributing to them in the future. For now, it is more important that I learn some programming skills.

Bye.

**TL;DR: I will no longer be a Discord bot developer. I still want to develop software though.**

**Comment by XInTheDark** [comment]
I'm currently returning to this project as some people have asked me to update the bot. However, I am willing to hand over this project to people who have time and energy to maintain it.
